gradle.properties

```diff
@@ -16,6 +16,6 @@ org.gradle.jvmargs=-Xmx1536m
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
Not sure if we need this or not for Android 6 related issues... You think this is appropriate to remove?

Reply:
It was only converting workmanager to androidx, if there is a problem with the new versoins of workmanager that do not need jetifier I can revert this

Reply:
Just spoke with husain, it looks like jetifier fixed many of the androidx-related issues coming from dependencies (outside of our control) on non-Pixel phones... but lets see how the next versions pans out.